dev-packages/e2e-tests/test-applications/nestjs-basic/tests/span-decorator.test.ts

```diff
@@ -1,15 +1,15 @@
 import { expect, test } from '@playwright/test';
 import { waitForTransaction } from '@sentry-internal/test-utils';
 
-test('Transaction includes span and correct value for decorated async function', async ({ baseURL }) => {
-  const transactionEventPromise = waitForTransaction('nestjs', transactionEvent => {
+test('Transaction includes span and correct value for decorated async function', async ({ request }) => {
```

Comment:
I: Here is one request left. Seems to work nevertheless but probably nicer to be consistent.